mobile-widgets/qmlinterface.cpp

```diff
@@ -20,4 +20,20 @@ void QMLInterface::setup(QQmlContext *ct)
 	// relink signals to QML
 	connect(qPrefCloudStorage::instance(), &qPrefCloudStorage::cloud_verification_statusChanged,
 		[=] (int value) { emit instance()->cloud_verification_statusChanged(CLOUD_STATUS(value)); });
+	connect(qPrefUnits::instance(), &qPrefUnits::duration_unitsChanged,
+			[=] (int value) { emit instance()->duration_unitsChanged(DURATION(value)); });
+	connect(qPrefUnits::instance(), &qPrefUnits::lengthChanged,
+			[=] (int value) { emit instance()->lengthChanged(LENGTH(value)); });
```

Comment:
I sense inconsistency here - and it seems that we already had this before...
`duration_unitsChanged()` vs `lengthChanged()`
and reading through the code it's even more confusing. The one that is marked as `...unitsChanged` is one that in fact changes a display preference (always show hours), and the others are the ones that really change units.

Not pretty. Actually, rather confusing. Bad maintainer / review work back then.

Out of scope for this PR for sure.